matter_server/server/device_controller.py

```diff
@@ -131,8 +131,21 @@ async def start(self) -> None:
             self._nodes[node_id] = node
             # setup subscription and (re)interview as task in the background
             # as we do not want it to block our startup
-            asyncio.create_task(self._check_interview_and_subscription(node_id))
-        LOGGER.debug("Loaded %s nodes", len(self._nodes))
+            if not node_dict.get("available"):
+                # if the node was not available last time we will delay
+                # the first attempt to initialize so that we prioritize nodes
+                # that are probably available so they are back online as soon as
+                # possible and we're not stuck trying to initialize nodes that are offline
+                assert self.server.loop
+                self.server.loop.call_later(
+                    5,
+                    asyncio.create_task,
+                    self._check_interview_and_subscription(node_id),
+                )
```

Comment:
It's better to have a callback that creates the task so that we don't have a non awaited coroutine possibly lingering if the server is shutdown before the delay.